### PR TITLE
Fix two bugs (see commit descriptions).

### PR DESCRIPTION
**Pull request by beorn7**
@juliusv @grobie @brian-brazil

### CODE REVIEW

File: prometheus/summary_test.go [hunk 1]
```diff
@@ -289,6 +289,11 @@ func TestSummaryVecConcurrency(t *testing.T) {
 }
 
 func TestSummaryDecay(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping test in short mode.")
+		// More because it depends on timing than because it is particularyl long...
```
Comment (juliusv) [inline prometheus/summary_test.go]: s/particularyl/particularly/

**Comment by juliusv** [comment]
:+1:

**Comment by brian-brazil** [comment]
:+1:

**Comment by grobie** [comment]
:+1: Thanks!

**Pull request merged**